docs/File-Formats.md

```diff
@@ -777,22 +777,21 @@ RET<TAB>5979<TAB>center.edu<TAB>SAMPLE_ID_3<TAB>Fusion<TAB>unknown<TAB>yes<TAB>u
 
 ## Case Lists
 
-There should be 1 or more case lists associated with each cancer study. You should provide **at least one case list which contains all sample ids** (the importer can generate this for your if you set the attribute *add_global_case_list* to 'true' in the [Study metadata](#cancer-study). 
+There should be 1 or more case lists associated with each cancer study. You should provide **at least one case list which contains all sample ids** (the importer can generate this for your if you set the attribute `add_global_case_list` to `true` in the [Study metadata](#cancer-study).
```

Comment:
maybe we should also mention that some case lists are mandatory if specific data types are present? Like `_sequenced` when there is mutation data.